ActivityListener/Domain/ActivityHistoryEntity.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace ActivityListener.Domain
+{
+    public class ActivityHistoryEntity
+    {
+        public Guid Id { get; set; }
+
+        public ActivityType Type { get; set; }
+
+        public string SourceDomain { get; set; }
+
+        public TargetType TargetType { get; set; }
+
+        public Guid TargetId { get; set; }
+
+        public DateTime CreatedAt { get; set; }
+
+        public int TimetoLiveForRecord { get; set; }
+
+        public object OldData { get; set; }
```

Comment:
Should not be `Dictionary<string, object>`?

Reply:
In my case here it doesn't need to be and it makes the listener tests easier because I don't have to force things into a specific typed object just for testing purposes.